.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,27 +1,28 @@
 Please work through the following checklists. Delete anything that isn't relevant.
 ## Development for new xarray-based metrics
-[] Works with n-dimensional data and includes `reduce_dims`, `preserve_dims`, and `weights` args.
-[] Typehints added
-[] Docstrings complete and followed Napoleon (google) style
-[] Reference to paper/webpage is in docstring
-[] Add error handling
-[] Imported into the API
+- [ ] Works with n-dimensional data and includes `reduce_dims`, `preserve_dims`, and `weights` args.
+- [ ] Typehints added
+- [ ] Docstrings complete and followed Napoleon (google) style
+- [ ] Reference to paper/webpage is in docstring
+- [ ] Add error handling
+- [ ] Imported into the API
 
 ## Testing of new xarray-based metrics
-[] 100% unit test coverage
-[] Test that metric is compatible with dask.
-[] Test that metrics work with inputs that contain NaNs
-[] Test that broadcasting with xarray works
-[] Test both reduce and preserve dims arguments work
-[] Test that errors are raised as expected
-[] Test that it works with both `xr.Dataarrays` and `xr.Datasets`
+- [ ] 100% unit test coverage
+- [ ] Test that metric is compatible with dask.
+- [ ] Test that metrics work with inputs that contain NaNs
+- [ ] Test that broadcasting with xarray works
+- [ ] Test both reduce and preserve dims arguments work
+- [ ] Test that errors are raised as expected
+- [ ] Test that it works with both `xr.Dataarrays` and `xr.Datasets`
 
 ## Tutorial notebook 
-[] Short introduction to why you would use that metric and what it tells you
-[] A link to a reference
-[] A "things to try next" section at the end
-[] Optional - a detailed discussion of how the metric works at the end of the notebook
+- [ ] Short introduction to why you would use that metric and what it tells you
+- [ ] A link to a reference
+- [ ] A "things to try next" section at the end
+- [ ] Add notebook to [Aplnation.ipynb](https://github.com/nci/scores/blob/develop/tutorials/Explanation.ipynb)
```

Comment:
"[Aplnation.ipynb]" I am guessing should be "[Explanation.ipnyb]"

Note: I don't know what to do when commenting on a closed pull request. Will this be seen? Should I open a new issue?